docs/reference/authentication.md

```diff
@@ -28,6 +28,14 @@ Web UI at Administration / Users to create users.  Users have an access key
 to log into the Web UI, or to authenticate programmatic requests to the API
 Server or the S3 Gateway.
 
+The built-in authentication database supports authentication to the UI, via
+email and password as well. This option is only valid for users who have 
+been invited to lakefs, confirmed the invite, and created a password.
+These credentials should be used for UI access only. It should be noted, that 
+the initial admin user who created the repo, will not be able to use email/password 
+authentiaction.
+
+
```

Comment:
1. For this to work, you need to enable an email server integration, right?
2. In general, note that this feature is interesting to both the system admin (the one who installs lakeFS), and to the end user. You should be more mindful of this.
3. "the admin user who created the repo" >> "the initial lakeFS admin user" - the user didn't necessarily create a repo.
4. Also see the [documentation guidelines](https://docs.google.com/document/d/1vif05nDiKZYPwYdINLoq4BJ6j7bBIQ6ZA2dt7McuzyY/edit): it has some helpful suggestions on which language to use. In particular, the branch name should be spelled as _lakeFS_.

Reply:
+1 for 1.
I think it's important to mention that using the email/password authentication requires additional configuration (and either provide a link to this configuration or explain it inline, IMO link to the full documentation is preferred).